src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py

```diff
@@ -229,6 +229,72 @@ def deleteConfigDoc(self, docType, docName):
         else:
             return self["requests"].delete('%sconfig/%s' % (docType, docName))[0]['result']
 
+    def getTransferInfo(self):
+        """
+        Get transfer document from coudhdb backend. The document has the following form:
+        {
+          "wf_A": [record1, record2, ...],
+          "wf_B": [....],
+        }
+        where each record has the following format:
+        {"timestamp":000, "dataset":"/a/b/c", "campaignType": "primary", "subscription": subscription}
+        subscription here either represent PhEDEx or Rucio subscriptions.
```

Comment:
`subscription here represents either a PhEDEx subscription ID or a Rucio rule ID`

Reply:
again please provide APIs which we should use and then we can adjust the docstring.